config/op-node/rollup_devnet.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
How is this file generated?

Reply:
It's mostly copy-pasted from [901-rollup.json](https://github.com/EspressoSystems/optimism-espresso-integration/blob/1c9f7087dcec889c33216f1e698919d054f94ba8/op-program/chainconfig/test/configs/901-rollup.json), with some unused info skipped.